src/components/editor/ResourceTemplateForm.jsx

```diff
@@ -61,7 +61,7 @@ export class ResourceTemplateForm extends Component {
 
   renderComponentForm = () => (
     <div>
-      <form>
+      <form onSubmit={e => e.preventDefault()}>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault

Basically, the default behavior of the "enter" key is "submit the form".  This prevents that from happening, because we aren't prepared to accept a standard HTML form submission.

Reply:
yeah, that's the doc I looked at.  I didn't understand how it related to the onBlur stuff -- is it part of that, or something else?